tests/unittests/tests-gnrc_ipv6_nib/tests-gnrc_ipv6_nib-pl.c

```diff
@@ -377,6 +377,46 @@ static void test_nib_pl_del__success(void)
     TEST_ASSERT(!gnrc_ipv6_nib_pl_iter(0, &iter_state, &ple));
 }
 
+/**
+ * Creates three prefix list entries and removes the second one
+ * The prefix list is then iterated.
+ * Expected result: there should be two prefix list entries returned, the first
+ * and the third one
+ */
+static void test_nib_pl_iter__empty_in_the_middle(void)
```

Comment:
Why test here only the removal of the middle entry and not also the removal of the first as with the routes?

Reply:
I only tested the removal of the first one with the routes because for default routes I have no control which one is removed (which I realize might be an issue iff people have multiple default routes [which on an embedded system I would highly discourage anyway], but this is not the topic of this PR)

Reply:
Ok